src/java/org/apache/cassandra/db/filter/ColumnFilter.java

```diff
@@ -295,14 +297,16 @@ public static ColumnFilter selection(TableMetadata metadata,
     }
 
     /**
-     * The columns that needs to be fetched internally for this filter.
+     * The columns that needs to be fetched internally.  See FetchingStrategy for why this is almost
+     * always a superset of the queried columns.
```

Comment:
When is it not a superset? The javadoc for ColumnFilter says `the _queried_ columns set is always included in the _fetched_ one.`. Are we changing that in this PR?

Reply:
You're right, I was sloppy in my wording.